common/network-common/src/test/java/org/apache/spark/network/RpcIntegrationSuite.java

```diff
@@ -207,9 +400,67 @@ private void assertErrorsContain(Set<String> errors, Set<String> contains) {
           break;
         }
       }
-      assertTrue("Could not find error containing " + contain + "; errors: " + errors, foundMatch);
+      if (!foundMatch) {
+        notFound.add(contain);
+      }
+    }
+    return new ImmutablePair<>(remainingErrors, notFound);
+  }
+
+  private static class VerifyingStreamCallback implements StreamCallbackWithID {
+    final String streamId;
+    final StreamSuite.TestCallback helper;
+    final OutputStream out;
+    final File outFile;
+    VerifyingStreamCallback(String streamId) throws IOException {
```

Comment:
nit: add empty line

Reply:
ping

Reply:
whoops, sorry I missed this one.  fixed now